spec/mailers/shared_examples/renders_proper_email_headers.rb

```diff
@@ -1,11 +1,20 @@
 RSpec.shared_examples "#renders_proper_email_headers" do
+  let(:from_email_address) { "custom_noreply@forem.com" }
+  let(:reply_to_email_address) { "custom_reply@forem.com" }
+
+  before do
+    allow(Settings::SMTP).to receive(:provided_minimum_settings?).and_return(true)
+    allow(Settings::SMTP).to receive(:from_email_address).and_return(from_email_address)
+    allow(Settings::SMTP).to receive(:reply_to_email_address).and_return(reply_to_email_address)
+  end
+
   it "renders proper sender" do
```

Comment:
Quick, non-blocking suggestion (same as above!): could we please add `:aggregate_failures` here since there are a handful of expectations that could fail within this one block?
```suggestion
  it "renders proper sender", :aggregate_failures do
```